Procfile

```
@@ -1,3 +1,3 @@
 web: newrelic-admin run-program gunicorn metadeploy.wsgi:application --preload
-worker: python manage.py rqworker default short
-scheduler: python manage.py rqscheduler default short
+worker: newrelic-admin run-python manage.py rqworker default short
+scheduler: newrelic-admin run-python manage.py rqscheduler default short
```

These commands seem to work, but I'm not sure how to test them functionally.

---

if you load `/admin/django-rq/` while the worker is up it should appear for each queue in the workers column.

sadly best way to check for scheduler atm is checking redis for the key, or try to bring up another scheduler instance and let it fail.

i'd love to get a PR back into django-rq someday to add scheduler info to the monitoring page but it hasn't been a priority...